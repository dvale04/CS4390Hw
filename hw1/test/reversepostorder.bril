@main {
  jmp .A;

.A:
  br true .B .C;

.B:
  jmp .D;

.C:
  jmp .D;

.D:
  ret;
}

