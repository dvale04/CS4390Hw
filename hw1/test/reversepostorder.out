digraph main {
 b0;
 A;
 B;
 C;
 D;
 b0 -> A;
 A -> B;
 A -> C;
 B -> D;
 C -> D;
}
